exoscale/datasource_exoscale_domain_record_test.go

```diff
@@ -0,0 +1,187 @@
+package exoscale
+
+import (
+	"errors"
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+var (
+	testAccDataSourceDomainRecordDomainName = testPrefix + "-" + testRandomString() + ".net"
+	testAccDataSourceDomainRecordName1      = "mail1"
+	testAccDataSourceDomainRecordName2      = "mail2"
+	testAccDataSourceDomainRecordType       = "MX"
+	testAccDataSourceDomainRecordContent1   = "mta1"
+	testAccDataSourceDomainRecordContent2   = "mta2"
+	testAccDataSourceDomainRecordPrio       = 10
+	testAccDataSourceDomainRecordTTL        = 10
+
+	testAccDataSourceDomainRecordConfigCreate1 = fmt.Sprintf(`
+resource "exoscale_domain" "exo" {
+  name = "%s"
+}
+
+resource "exoscale_domain_record" "mx" {
+  domain      = "${exoscale_domain.exo.id}"
```

Comment:
Please update your configuration syntax to use TF 0.12+ format (also applies for the documentation examples below).